packages/api-client/src/markup/types.ts

```diff
@@ -21,6 +21,24 @@ export type MarkupRef = CollaborativeDoc
 /** @public */
 export type MarkupFormat = 'markup' | 'html' | 'markdown'
 
+/** @public */
+export class MarkupContent {
+  constructor (
+    readonly content: string,
+    readonly kind: MarkupFormat
+  ) {}
+}
+
+/** @public */
+export function html (content: string): MarkupContent {
```

Comment:
Do we really need html?

Reply:
Yes, why not. HTML is still very popular and native for some editors, including tiptap. It is easier to convert from/to HTML than markdown.